server/routes/stock-details.js

```diff
@@ -6,6 +6,33 @@ const defaultData = {
     stock_details: {}
```

Comment:
can we delete defaultData?

Reply:
I would say its better to have it there to prevent duplicate code